src/app/lib/access/bugsee/bugsee_repository.dart

```diff
@@ -13,18 +13,39 @@ abstract interface class BugseeRepository {
 
   /// Update the current video captured or not flag in shared prefs.
   Future setIsVideoCaptureEnabled(bool isVideoCaptureEnabled);
+
+  /// Update whether data is obscure in shared prefs.
+  Future setIsDataObscure(bool isDataObscure);
+
+  /// Update the logCollection flag in shared prefs.
+  Future setIsLogCollectionEnabled(bool isLogCollectionEnabled);
+
+  /// Update the logFilter flag in shared prefs.
+  Future setIsLogFilterEnabled(bool isLogFilterEnabled);
+
+  /// Update the attachFile boolean flag in shared prefs.
+  Future setAttachLogFileEnabled(bool attachLogFile);
 }
 
 final class _BugseeRepository implements BugseeRepository {
   final String _bugseeEnabledKey = 'bugseeEnabledKey';
   final String _videoCaptureKey = 'videoCaptureKey';
+  final String _dataObscureKey = 'dataObscureKey';
```

Comment:
I think all those shared preference keys should have Bugsee written in them so that they don't conflict with potential app features.